crates/modelardb_server/tests/integration_test.rs

```diff
@@ -736,3 +761,27 @@ fn test_optimized_query_results_equals_non_optimized_query_results() {
 
     assert_eq!(optimized_query, non_optimized_query);
 }
+
+#[test]
+fn test_cannot_update_invalid_setting() {
```

Comment:
Maybe replace `invalid` with `non_existing` to make it clear that it does not exist.

Reply:
Fixed.